gradle.properties

```diff
@@ -15,6 +15,7 @@ org.gradle.parallel=true
 kotlin.code.style=official
 # Enable R8 full mode.
 org.gradle.unsafe.configuration-cache=false
+org.gradle.daemon=true
```

Comment:
That's not important although it doesn't seem to be working correctly. It suppose to leave the gradle daemon and therefore it should speed up the build because it won't need to start over and over again.

Reply:
I see. Thanks.